autoscheduler/frontend/src/components/GenericCard/GenericCard.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react';
+import * as styles from './GenericCard.css';
+
+interface GenericCardProps {
+    header?: JSX.Element;
+    style?: React.CSSProperties;
+}
+
+/**
+ * Makes a generic card with a maroon header. Any children are rendered inside the body, and
+ * elements can be rendered inside the header by setting the header prop
+ * @param props Includes header, children, and minWidth, all optional
```

Comment:
```suggestion
 * @param props Includes header and children, which are optional
```
Something like this, since `minWidth` isn't a  parameter

Reply:
oh lmao i forgot that i changed this